tests/phpunit/CRMTraits/Financial/FinancialACLTrait.php

```diff
@@ -20,24 +20,18 @@ trait CRMTraits_Financial_FinancialACLTrait {
    * Enable financial ACLs.
    */
   protected function enableFinancialACLs() {
-    $contributeSettings = Civi::settings()->get('contribution_invoice_settings');
     $this->callAPISuccess('Setting', 'create', [
-      'contribution_invoice_settings' => array_merge($contributeSettings, ['acl_financial_type' => TRUE]),
```

Comment:
this was actually the first setting we separated out of the contribution_invoice_settings borg & has been a free agent even longer than the rest